i18n-unused.config.js

```diff
@@ -0,0 +1,19 @@
+// Info: https://www.npmjs.com/package/i18n-unused
+
+module.exports = {
+  localesPath: 'packages/common/src/intl/locales',
+  srcPath: 'packages',
+  ignorePaths: [
+    'packages/common/node_modules',
+    'packages/android/node_modules',
+    'packages/config/node_modules',
+    'packages/dashboard/node_modules',
+    'packages/host/node_modules',
+    'packages/inventory/node_modules',
+    'packages/invoices/node_modules',
+    'packages/requisitions/node_modules',
+    'packages/system/node_modules',
+    'packages/template/node_modules',
```

Comment:
```suggestion
    'packages/common/node_modules',
    'packages/config/node_modules',
    'packages/dashboard/node_modules',
    'packages/host/node_modules',
    'packages/host/dist',
    'packages/inventory/node_modules',
    'packages/invoices/node_modules',
    'packages/requisitions/node_modules',
    'packages/system/node_modules',
    'packages/template/node_modules',
    'packages/android',
    'packages/android/node_modules',
    '../node_modules',
```

ok - so I tried running `i18n-unused display-missed` and that has highlighted the need for the ignore prop, and some bits which will need adding..
Still disappointed that we have to be explicit with all the paths, but I'll just have to live with disappointment.